README.md

```diff
@@ -1,7 +1,7 @@
 # cri
 <p align="center">
 <img src="https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png" width="50" height="50">
-<img src="https://github.com/containerd/containerd/blob/master/docs/images/containerd-dark.png" width="200" >
+<img src="https://github.com/containerd/containerd/blob/v1.1.0/docs/images/containerd-dark.png" width="200" >
```

Comment:
it moved to the docs/static/img directory... we should use  "https://github.com/containerd/containerd/blob/master/docs/static/img/containerd-dark.png"

Reply:
If we use master, we may still have the problem in the future.
Let's use a tagged release, so it won't change.